skrub/_joiner.py

```diff
@@ -144,11 +249,28 @@ def fit(self, X: pd.DataFrame, y=None) -> "Joiner":
         Joiner
             Fitted Joiner instance (self).
         """
+        del y
         self._main_key, self._aux_key = _join_utils.check_key(
             self.main_key, self.aux_key, self.key
         )
         _join_utils.check_missing_columns(X, self._main_key, "'X' (the main table)")
         _join_utils.check_missing_columns(self.aux_table, self._aux_key, "'aux_table'")
+        self._check_max_dist()
+        self.vectorizer_ = _make_vectorizer(
+            self.aux_table[self._aux_key],
+            self.string_encoder,
+            rescale=self.ref_dist != "raw_distance",
```

Comment:
I wonder if it wouldn't be a bit redundant because it adds an attribute that computes something trivial and does not have a direct correspondence in the parameters so users have to discover and understand what `rescale_` means when they can already check if they passed `ref_dist="no_rescaling"` (btw raw_distance has been renamed no_rescaling)